index.html

```diff
@@ -3,89 +3,81 @@
-        <div>
-            <button class="btn">READ MORE</button>
-        </div>
+   </div>  
+
+   <div class="img-section">
+      <div>
+       <p>The home for your favorite books!</p>
+       <button>READ MORE</button>
+     </div>
     </div>
 
-    <div class="books">
-        <div class="book">
-            <div class="title">The Great Gatsby</div>
-            <div class="author">F. Scott Fitzgerald</div>
-        </div>        
-        <div class="book">
-            <div class="title">The Sound and the Fury</div>
-            <div class="author">William Faulkner</div>
-        </div>        
-        <div class="book">
-            <div class="title">The Name of the Rose</div>
-            <div class="author">Umberto Eco</div>
-        </div>        
-        <div class="book">
-            <div class="title">The Magic Mountain</div>
-            <div class="author">Thomas Mann</div>
-        </div>        
-        <div class="book">
-            <div class="title">In Search of Lost Time</div>
-            <div class="author">Marcel Proust</div>
-        </div>        
-        <div class="book">
-            <div class="title">To the Lighthouse</div>
-            <div class="author">Virginia Woolf</div>
-        </div>        
-        <div class="book">
-            <div class="title">Nineteen Eighty-Four</div>
-            <div class="author">George Orwell</div>
-        </div>        
-        <div class="book">
-            <div class="title">The Hound of the Baskervilles</div>
-            <div class="author">Arthur Conan Doyle</div>
-        </div>        
-        <div class="book">
-            <div class="title">Ulysses</div>
-            <div class="author">James Joyce</div>
-        </div>        
-        <div class="book">
-            <div class="title">The Little Prince</div>
-            <div class="author">Antoine de Saint-Exupéry</div>
-        </div>        
-        <div class="book">
-            <div class="title">The Name of the Rose</div>
-            <div class="author">Umberto Eco</div>
-        </div>        
-        <div class="book">
-            <div class="title">The War of the Worlds</div>
-            <div class="author">H.G. Wells</div>
-        </div>        
-        <div class="book">
-            <div class="title">One Hundred Years of Solitude</div>
-            <div class="author">Gabriel García Márquez</div>
-        </div>        
-        <div class="book">
-            <div class="title">Waiting for Godot</div>
-            <div class="author">Samuel Beckett</div>
-        </div>        
-        <div class="book">
-            <div class="title">Gone with the Wind</div>
-            <div class="author">Margaret Mitchell</div>
-        </div>        
-        <div class="book">
-            <div class="title">Friday</div>
-            <div class="author">Michel Tournier</div>
-        </div>
+    <div class="last-section">
```

Comment:
This are is supposed to be loaded from the API and the JSON object should be a book list with authors and title.
Here I can't see the structure and how it should be evolved to the API. It is a step back from not towards that.